src/api/spec/components/previews/bs_request_action_description_component_preview.rb

```diff
@@ -7,6 +7,8 @@ def submit_preview
 
   def add_role_preview
     bs_request = BsRequest.joins(:bs_request_actions).where(bs_request_actions: { type: :add_role }).last
-    render(BsRequestActionDescriptionComponent.new(action: bs_request.webui_actions.find { |a| a[:type] == :add_role }, creator: bs_request.creator))
+    webui_action = bs_request.webui_actions.find { |a| a[:type] == :add_role }
```

Comment:
This is a `.where` and there can be more than one `add_role` action.

Reply:
I simplified everything